### ISSUE TITLE
Installer should fall back to non privileged mode

### ISSUE THREAD

**Issue by ferrybig** [ShareX/ShareX#3227]
OS: WIndows 10 home

1. Setup your pc with an admin account and a non admin account
2. Install ShareX using the admin account
3. Use ShareX from your normal account
4. Wait till ShareX releases a update
5. Install the update if ShareX asks it

Expected:

6. Either ShareX starts back as my normal account, or it doesn't start after instlation

Actual:

6. ShareX starts as my admin account, and has lost all settings, until restarted under my normal account

**Comment by Muramas** [comment]
I would also like to point out that when you install sharex it doesn't prompt for admin if UAC is enabled. You need to select "run as admin" for it to work.

**Comment by Jaex** [comment]
Duplicate of #1267

I couldn't fix it.

**Comment by ferrybig** [comment]
I see you spend a lot of time on this already, and I appreciate this.

Would it be hard to add a checkbox after upgrading to automatically start the program? That way people who want to use the application in a secure way can uncheck it and manually start the program. instead of clicking through all startup warnings like conflicting keys, and quit it, before starting the program.
